src/main/User/Services/GetMembersService.java

```diff
@@ -155,4 +111,38 @@ private static JSONArray getPage(JSONArray elements, int pageStartIndex, int pag
     }
     return page;
   }
+
+  // Return 1 if added to list to keep total below threshold
+  private int constructUserList(User user, ListType listType, JSONArray userList) {
+    JSONObject userJSON = new JSONObject();
+    userJSON.put("username", user.getUsername());
+    userJSON.put("privilegeLevel", user.getUserType());
+    userJSON.put("firstName", user.getFirstName());
+    userJSON.put("lastName", user.getLastName());
+    userJSON.put("email", user.getEmail());
+    userJSON.put("phone", user.getPhone());
+    userJSON.put("address", user.getAddress());
+    userJSON.put("city", user.getCity());
+    userJSON.put("state", user.getState());
+    userJSON.put("zipcode", user.getZipcode());
+    UserType userType = user.getUserType();
```

Comment:
if we keep having to do this a lot, why dont we add a method to the User class called serialize() which would just return a JSON object so in the future if we have to add new fields we dont have to remember to manually go to each of these places and update the fields?

Reply:
Great idea, I’ll add that